### ISSUE TITLE
Make card colors configurable

### ISSUE THREAD

**Issue by AitorDB** [AitorDB/home-assistant-sun-card#21]
Card colors, specially the graph ones, could be configurable so it can be adapted to each dashboard theme

Context: https://www.reddit.com/r/homeassistant/comments/ne2mx4/sun_card_for_home_assistant/gyeip4x

**Comment by NiMa08** [comment]
I would like this as well. My theme is dark, but the text color stays black while the rest of my text is white.

**Comment by TheLordVader** [comment]
Yes! I have the same issue, text color stays black when dark theme.

**Comment by rjenx** [comment]
Me too, great card by the way! Thank you

**Comment by LiJu09** [comment]
To make color of text white add `darkMode: true` to card configuration.

**Comment by rjenx** [comment]
For me this is now closed, the suggestion by [LiJu09](https://github.com/LiJu09) worked for me.
